### PR TITLE
[media] Setup code formatting for //media/starboard

### PR DESCRIPTION
**Pull request by xiaomings**
1. Add //media/starboard/ to .github/CODEOWNERS and .pre-commit-config.yaml so any files in it will be re-formatted during pre-commit.
2. Format all files in it, without any other modifications, to set them up for future changes.

b/328305706

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,6 @@
 * @youtube/cobalt-3p-repository-owners
 /.github/  # no owners
 /cobalt/  # no owners
+/chrobalt/  # no owners
```
Comment (kaidokert) [inline .github/CODEOWNERS:4]: This one should be removed now ! Brian renamed chrobalt->cobalt

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -9,6 +9,8 @@ files: |
     (?x)^(
         .github/|
         cobalt/|
+        chrobalt/|
```
Comment (kaidokert) [inline .pre-commit-config.yaml:12]: Remove , see above

File: .github/CODEOWNERS (hunk 1 above)
Comment (xiaomings) [reply .github/CODEOWNERS]: Done

File: .pre-commit-config.yaml (hunk 2 above)
Comment (xiaomings) [reply .pre-commit-config.yaml]: Done

**Review by jasonzhangxx — APPROVED** [review]

**Review by kaidokert — APPROVED** [review]

**Pull request merged**